plottable.d.ts

```diff
@@ -3680,15 +3692,26 @@ declare module Plottable {
 
 declare module Plottable {
     class Interaction {
+        protected _componentAttachedTo: Component;
+        protected _anchor(component: Component): void;
+        protected _unanchor(): void;
         /**
-         * It maintains a 'hitBox' which is where all event listeners are
-         * attached. Due to cross- browser weirdness, the hitbox needs to be an
-         * opaque but invisible rectangle.  TODO: We should give the interaction
-         * "foreground" and "background" elements where it can draw things,
-         * e.g. crosshairs.
+         * Attaches current interaction to a Component. If the interaction was already
+         * attached to a Component, it first detaches itself from the old component.
+         *
+         * @param {Component} component The component to which to attach the interaction.
+         *
+         * @return {Interaction}
+         */
+        attachTo(component: Component): Interaction;
+        /**
+         * Detaches current interaction from the Component. Interaction can be reused.
```

Comment:
"This interaction can be reused." and also consider placing this on a second line